ako-operator/bundle/manifests/ako-operator.clusterserviceversion.yaml

```diff
@@ -37,12 +37,13 @@ metadata:
             "controllerSettings": {
               "cloudName": "Default-Cloud",
               "controllerIP": "",
-              "controllerVersion": "20.1.5",
+              "controllerVersion": "22.1.2",
               "serviceEngineGroupName": "Default-Group",
               "tenantName": "admin"
             },
+            "replicaCount": 1,
             "imagePullPolicy": "IfNotPresent",
-            "imageRepository": "projects.registry.vmware.com/ako/ako@sha256:c900213eea87e6217d751aa3d130c594abe1d794d8ce1aff757c77a57165639e",
+            "imageRepository": "projects.registry.vmware.com/ako/ako@sha256:f80d2af984f76e176f441190fa2ee97cc0227f880a27361fc4d5405741bb729a",
```

Comment:
This `sha` is of 1.9.1. We should have sha of 1.9.3

Reply:
Have not made the bundle changes yet. Will add it in the next PR with updated `sha` for ako-operator:1.9.3 image once this PR is merged and image is generated.